_scripts/templates/Go/tester.psm1

```diff
@@ -1,14 +1,20 @@
 # Template generated code from trgen <version>
 function Build-Grammar {
-<tool_grammar_files:{x |
-    $g = antlr <x> -Dlanguage=Go <antlr_tool_args:{y | <y> } >
+<tool_grammar_tuples:{x |
+    $g = antlr <x.GrammarFileName> -encoding <antlr_encoding> -Dlanguage=Go <x.AntlrArgs> <antlr_tool_args:{y | <y> } >
```

Comment:
Some of the attributes were changed. I now have all the grammars in a set of tuples, which describe the name of the grammar, the location of the grammar file in the original grammars-v4 directory, the target location (in "Generated/"), the name of the function to call to create the parser, etc. Also, I notice that the tester.psm1 script wasn't setting the encoding of the input grammar. Most of the time this is empty, but there are a couple of grammars in grammars-v4 where it is not. For "Go", I added "AntlrArgs" which is a per-grammar set of options. Unfortunately, Windows and Linux differ, where -o/-lib are required for Windows, and nothing for Linux. I noted this problem in Antlr, and someone recently ran across the problem, but likely will never be fixed.